docs/work-with-reports/HistoryOfLaunches.mdx

```diff
@@ -12,44 +12,26 @@ the most unstable test cases in your suites/ launches/ builds.
 
 To see a table of the historical trend of executions, navigate to the "Launches" page and drill down to launch structure/ or launch statistics.
 
-### History table: Test Case ID or Unique ID
+### History table based on Test Case Hash
 
-ReportPortal acquires a lot of real-time test results and you can choose by which parameter you want the history of test results to get defined.
-There are 2 options:
+Test case hash is a parameter that is automatically generated based on the Test Case ID (if specified) or the full path to the test.
 
-* by [Unique ID](./UniqueId) ***(deprecated/ will be replaced by Test Case ID gradually)***
-* by [Test Case ID](./TestCaseId) 
-
-### Difference between Test Case ID  and Unique ID
-
-Please find the description of what Test Case ID and Unique  by the links:
-* [UUID](./UniqueId) ***(deprecated/ will be replaced by Test Case ID gradually)***
-* [Test Case ID](./TestCaseId) 
-
-They are both a unique identifiers of executions uniquness.
-
-But Unique ID ***(deprecated)*** generates on the base of a launch name where test execution is situated.
-And Test Case ID usually doesn't contain a launch name. 
-
-So that the main difference between a history table based on Unique ID ***(deprecated)*** and Test Case ID is that:
-
-* History on Unique ID ***(deprecated)*** shows only executions from launches with the same name
-* History on Test Case ID shows executions from all launches.
+You can read about the Test Case ID [here](/work-with-reports/TestCaseId).
 
 ### Unique ID history
-***(deprecated/ will be replaced by Test Case ID gradually)***
+***(deprecated)***
 
 1. Take the ReportPortal release version 5.2.2 or higher. docker-compose.yml
 
 2. Add an environment variable to the service-API service:
 
 ```
-RP_ENVIRONMENT_VARIABLE_HISTORY_OLD=true
+RP_ENVIRONMENT_VARIABLE_HISTORY_OLD=false
```

Comment:
Why this was changed? In this case you won't enable old history.

Reply:
@pressayuliya for 'Unique ID history' there should be RP_ENVIRONMENT_VARIABLE_HISTORY_OLD=true.
please change it back